charts/sourcegraph/templates/jaeger/jaeger.Deployment.yaml

```diff
@@ -53,7 +53,7 @@ spec:
       - name: jaeger
         image: {{ include "sourcegraph.image" (list . "jaeger") }}
         imagePullPolicy: {{ .Values.sourcegraph.image.pullPolicy }}
-        args: {{- default (list "--memory.max-traces=20000") .Values.jaeger.args | toYaml | nindent 8 }}
+        args: {{- default (list "--memory.max-traces=20000" "--sampling.strategies-file=/etc/jaeger/sampling_strategies.json" "--collector.otlp.enabled") .Values.jaeger.args | toYaml | nindent 8 }}
```

Comment:
Minor: does this mean if one provides their own args to the chart, they need to make sure to include the default args explicitly?

Reply:
Yeah it does, if they provide their own entry args these will be overwritten. Thanks for pointing it out, something for docs. Maybe worth it to add the args to the `values.yml` so they are recommended there as a default as well? cc: @michaellzc